### ISSUE TITLE
Is `import . "importpath"` supported ?

### ISSUE THREAD

**Issue by raff** [dave/jennifer#46]
I can see how to do regular imports and anonymous import ( import _ "importpath" )  but I am not sure of how to correctly code the case where package name is "."

If package name is "." the specs say:
> If an explicit period (.) appears instead of a name, all the package's exported identifiers declared in that package's package block will be declared in the importing source file's file block and must be accessed without a qualifier.

The behavior should be that if I, for example, import "fmt" with ".", then when I use Qual("fmt", "Println") the "fmt" package should be imported and the Qual() should expand to "Println").

Is this possible ? I have tried using ImportAlias("fmt", ".") but this ends up generating "..Println".

**Comment by dave** [comment]
Yes I don't think this is currently supported... I'll see if I can get something working over the next few days...

**Comment by raff** [comment]
Thanks!

**Comment by raff** [comment]
If you can give some suggestion on how to implement this properly, I could try to do it and submit a pull request.

**Comment by dave** [comment]
Hey sorry I haven't had time to look at this. I've been flat-out working on https://github.com/dave/dst... 

If you could do the change and submit a pull request that would be great. I think the right way to approach it is with `ImportAlias("fmt", ".")`. It may actually just need a small tweak here: https://github.com/dave/jennifer/blob/94ad458fe2628ada73d7c7ed0ca2ceae79a4fbea/jen/tokens.go#L34 ... 

... I think it also needs some documentation and tests.

**Comment by magicsong** [comment]
as at right there is no way to import ".", is there a way to force write some imports that doesn't have related `Qual` ?

**Comment by dave** [comment]
Right now I don't think there's a work-around for this, but it's a pretty simple fix so I'll see what I can come up with this week.

**Comment by dave** [comment]
This is fixed now. To add a dot-import, use `ImportAlias("<path>", ".")`.

I've released this as v1.2... Let me know if there's any problems!
